app/services/the_tvdb_service/row.rb

```diff
@@ -0,0 +1,52 @@
+class TheTvdbService
+  class Row
+    attr_reader :media_id, :media_type, :data, :episode_id, :tvdb_series_id
+
+    def initialize(media_id, data, tvdb_series_id)
+      @media_id = media_id
+      @data = data
+      @episode_id = data['id']
+      @tvdb_series_id = tvdb_series_id
+      @media_type = 'Anime'
+    end
+
+    def update_episode!
+      return unless episode_number.present?
+
+      episode.season_number ||= data['airedSeason']
+      episode.relative_number ||= data['airedEpisodeNumber']
+      # was returning an empty string and storing that in database.
```

Comment:
What was?  When?

Others reading this code will have none of the context you did when writing it, you need to remember that while writing comments